home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -137,9 +137,9 @@ async def async_setup(hass, config):
         hass.data[DATA_XKNX].async_create_exposures()
         await hass.data[DATA_XKNX].start()
     except XKNXException as ex:
-        _LOGGER.warning("Can't connect to KNX interface: %s", ex)
+        _LOGGER.warning("Cannot connect to KNX interface: %s", ex)
```

Comment:
Can we use `Could not` or what was the intention of changing this?

Reply:
"Could not" also works. I just wanted to get rid of the informal "Can't" and the typo "Cant" while I was already adjusting other messages.

Reply:
I personally don't care. If you want to change it again go ahead (I'd like that) but if not that's also no issue for me. In the end its the same.